### PR TITLE
Create AGP-Comprehensive-Aragon-Legal-Review-Program.md

### PR DESCRIPTION
**Pull request by burrrata**
Created a proposal to provide legal support to the Aragon ecosystem so that people can build, ship, and use Aragon apps with confidence.

### CODE REVIEW

File: AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+
```
Comment (izqui) [inline AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md]: Formatting: please remove extra new lines

File: AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Status: Stage III
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Aragon Comprehensive Legal Review Program
+
+## Description of desired Association policy change
+
+Fund 3-5 hours of legal council for every Aragon application shipped through the Nest or Flock program. This council would determine:
```
Comment (izqui) [inline AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md]: Up to 3-5h

**Comment by john-light** [comment]
The time now is after the published deadline of October 10, 2019 at 16:00 UTC for finalizing proposals and moving them to Stage V, and this proposal has not yet been finalized. Per [AGP-1](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-1.md#stage-iv-community-review), this proposal is considered to be withdrawn and this pull request will now be closed:

> If you do not finalize the proposal and request to move the proposal to Stage V before Stage V is scheduled to begin, AGP Editors will consider the proposal withdrawn and close the pull request when Stage V begins.

File: AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md (hunk 2 above)
Comment (burrrata) [reply AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md]: Good call. Thanks

Comment (burrrata) [reply AGPs/AGP-Comprehensive-Aragon-Legal-Review-Program.md]: New PR for this AGP can be found here:
- https://github.com/aragon/AGPs/pull/118

**Comment by burrrata** [comment]
New PR for this AGP
- https://github.com/aragon/AGPs/pull/118

**Pull request closed without merging**